theano/tensor/nnet/abstract_conv.py

```diff
@@ -422,7 +422,7 @@ def conv2d_grad_wrt_weights(input,
                                                filter_flip=filter_flip,
                                                filter_dilation=filter_dilation)
 
-    return gradWeight_op(input, output_grad, filter_shape[:-2])
```

Comment:
thanks.

Can you finish gh-4711? Is there something left in it?

On Mon, Aug 29, 2016 at 11:36 AM, ballasn <notifications@github.com> wrote:

> In theano/tensor/nnet/abstract_conv.py
> <https://github.com/Theano/Theano/pull/4911#discussion_r76629122>:
>
> > @@ -422,7 +422,7 @@ def conv2d_grad_wrt_weights(input,
> >                                                 filter_flip=filter_flip,
> >                                                 filter_dilation=filter_dilation)
> >
> > -    return gradWeight_op(input, output_grad, filter_shape[:-2])
>
> It needs a new test, but the test will be introduced with #4711
> <https://github.com/Theano/Theano/pull/4711> pull request.
>
> It fixes the issue #4885 <https://github.com/Theano/Theano/issues/4885>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4911/files/c50b91c4bb7b0aaa15d35010b5fedb806ab61091#r76629122>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AALC-6OKtXFTrXHeopKXW1GvsM9im2Xpks5qkvx2gaJpZM4JvlUr>
> .
>